tests/src/system/basic/WskBasicTests.scala

```diff
@@ -334,6 +334,17 @@ class WskBasicTests
             }
     }
 
+    it should "display a trigger summary when --summary flag is used with 'wsk trigger get'" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val triggerName = "mySummaryTrigger"
+            assetHelper.withCleaner(wsk.trigger, triggerName, confirmDelete = false) {
+                (trigger, name) => trigger.create(name)
+            }
+            // Summary namespace should match one of the allowable namespaces (typically 'guest')
+            val ns_regex_list = wsk.namespace.list().stdout.trim().replace('\n', '|')
```

Comment:
tip, `()` may be dropped on methods that accept no parameters - by convention, we drop these on methods that are not side-effecting, like `trim`.